### DISCUSSION TITLE
Unlabeled functions in 10.3 now default to unknown cc

### DISCUSSION THREAD

**Discussion by matcool** [NationalSecurityAgency/ghidra#5358] [Q&A]
Previously, when opening the edit function signature popup it would default to the calling convention that the decompiler guessed, but now in 10.3 it always defaults to unknown along with the `Use Custom Storage` option being enabled.
Is there any way to revert this behavior? The old behavior was more convenient when renaming functions since the guessed cc was often right

**Comment by astrelsky** [comment]
I'm going to assume this is related to #5316. Once it isn't able to guess correctly things seem to go pretty far south.

**Comment by ghidra1** [comment]
I believe I found the regression with the Decompiler's Edit Function action.  Will put fix for 10.3.1 patch release.  See #5367
